package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "meteor-user-status",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Since this is only updating dev dependencies I see no need to publish a new version.